daemon/src/engine/server/server.h

```diff
@@ -379,7 +379,7 @@ void SV_GetPlayerPubkey( int clientNum, char *pubkey, int size );
 void SV_CreateBaseline();
 
 void SV_ChangeMaxClients();
-void SV_SpawnServer( const char *server );
+void SV_SpawnServer(const std::string pakname, const std::string server);
```

Comment:
We need the pakname later, it is no longer `"map-" + name"`
Must be an owned string (at some point, so why not here) as the filesystem is cleared in this function invalidating any prior references